musig/README.md

```diff
@@ -34,19 +34,63 @@ Ristretto base point in compressed form:
 B = e2f2ae0a6abc4e71a884a961c500515f58e30b6aa582dd8db6a65945e08d2d76
 ```
 
-### Multikey
+### MusigContext
+
+This is a trait with two functions:
+- `commit(&self, &mut transcript)`: takes a mutable transcript, and commits the internal context to the transcript.
+- `challenge(&self, &verification_key, &mut transcript, &nonce_commitment) -> Scalar`: takes a public key and mutable transcript, and returns the 
```

Comment:
The addition of the `sum{R_i}` is the same for both contexts, so we can move this outside this context and perform inside the Party...::receive_commitments(), while the contexts are only doing context-specific things.

```suggestion
- `challenge(&self, &verification_key, &mut transcript) -> Scalar`: takes a public key and mutable transcript, and returns the 
```

Reply:
sounds good!